apps/data-types.md

```diff
@@ -641,6 +641,7 @@ This object is used for the attributes of basePrice, depositPrice, ecoParticipat
 | energyLabelsString | string | A list of energy labels applied to this product. Can be one value or a range with two values. |
 | energyLabelsMaxEfficiencyString | string | The highest possible efficiency class of an uploaded energy label. |
 | energyLabelSourceFile | string | A URL with an image or PDF file containing the energy label image supplied by the manufacturer. |
+| productDataSheetSourceFile | string | A URL with an image or PDF file containing the data sheet for the product. |
```

Comment:
To make the description consistent with `productDataSheet` within the product resources:
```suggestion
| productDataSheetSourceFile | string | A URL with an image or PDF file containing a data sheet with technical information on the product. Has to be available if the product has an energy label. |
```

Reply:
'Has to be available if the product has an energy label.'

I see what you mean, but this isn't really true, the productDataSheet is independent of the energy label stuff. Perhaps this quote was meant for 'energyLabelSourceFile' instead, that would actually make sense.

Reply:
@bolsu I see. But maybe we should then correct it and also add this information to the description of `productDataSheetSourceFile` for the data type productLineItem? So something around these lines: "Has to be available if `energyLabelSourceFile` is provided."
What do you think?

Reply:
@dkluever22 That line is incorrect for both Product / ProductLineItem, it should not be there for either. Have I misunderstood something?

Reply:
@bolsu No, it seems that it was a misunderstanding on our end. So you're saying that in the description of `productDataSheet` the statement "Has to be available if the product has an energy label." is incorrect. The only place where this sentence makes sense is within the description of `energyLabelSourceFile`. Correct?
So no dependencies between product data sheets and energy labels?
I'm just wondering why this information was put there in the first place and if it had anything to do with this ticket: https://epages.atlassian.net/browse/EPUI-1988

Reply:
@dkluever22 Now we're on the same page :)

I don't think it is to do with that issue, it was probably done a long time ago. We could probably find out if we cared enough to go through the file's history ...

Reply:
@bolsu Oooh, I see. Maybe when we will have a loooot of time we can check it out :D